how-to/workspace-platform-starter/client/src/framework/bootstrapper.ts

```diff
@@ -201,8 +202,22 @@ export async function init(): Promise<boolean> {
 		authProvider.subscribe("logged-in", async () => {
 			// what behavior do you want to do when someone logs in
 			// potentially the inverse if you hid something on session expiration
-			await fireLifecycleEvent(platform, "auth-logged-in");
+			await fireLifecycleEvent(platform, "auth-logged-in", {
```

Comment:
Maybe we need a flag here in the authProvider settings? Publish userInfo on loggedIn?

There may be a platform that wants to keep that info private and only expose it through their own means e.g. channel api from auth module etc. So we should give the option of whether or not that is published with the event.